cli/cmd/resources.go

```diff
@@ -271,6 +271,9 @@ func setupResources() {
 	extraHeaders := http.Header{}
 	extraHeaders.Set("x-client-id", analytics.ClientID())
 	extraHeaders.Set("x-source", "cli")
+	extraHeaders.Set("x-organization-id", cliConfig.OrganizationID)
```

Comment:
We are attaching the new headers as part of the request

Reply:
So we are always going to send this headers, right? even for OSS?